### PR TITLE
feat: introduce basic open API specification

### LINKED ISSUE #76: M3: BE: REST-API
- https://github.com/aztlan-labs/chicmoz/pull/17
- https://github.com/aztlan-labs/chicmoz/pull/25
- #78

### PR DESCRIPTION
**Pull request by FilipHarald**
resolves #76

### CODE REVIEW

File: services/explorer-api/src/http-server/open-api-spec.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "title": "Aztec Scan API",
+    "version": "0.2.0",
+    "description": "API for exploring Aztec Network. Please note that this is a work in progress and the API is subject to change.",
+  },
+  "servers": [
+    {
+      "url": "https://api.chicmoz.info/v1/{apiKey}",
+      "variables": {
+        "apiKey": {
+          "default": "d1e2083a-660c-4314-a6f2-1d42f4b944f4",
```
Comment (Mautjee) [inline services/explorer-api/src/http-server/open-api-spec.ts:15]: Do we keep this api this public??

File: services/explorer-api/src/http-server/routes/controllers/inofficial.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  address,
+  blockHash,
+  blockHeight,
+  heightOrHash,
+  paths,
+  txEffectIndex,
+  txHash,
+} from "../paths_and_validation.js";
+
+const SUB_PATH = "/v1/d1e2083a-660c-4314-a6f2-1d42f4b944f4";
```
Comment (Mautjee) [inline services/explorer-api/src/http-server/routes/controllers/inofficial.ts:13]: maybe make a const of this api key so if it chachges we do not have to go trough the code base

**Review by Mautjee — APPROVED** [review]

File: services/explorer-api/src/http-server/open-api-spec.ts (hunk 1 above)
Comment (FilipHarald) [reply services/explorer-api/src/http-server/open-api-spec.ts:15]: Yes, it's public in the frontend anyway. The reason is that we haven't implemented `auth`-service yet. But to keep the correct auth-flow we just hardcode it.

Comment (FilipHarald) [reply services/explorer-api/src/http-server/open-api-spec.ts]: https://github.com/aztlan-labs/chicmoz/issues/61

**Pull request merged**